### DISCUSSION TITLE
SKR Mini E3 V2.0 Compatibility

### DISCUSSION THREAD

**Discussion by raymond-arifianto** [Jyers/Marlin#861] [Q&A] [answered]
Hello, I never compiled a firmware on my own and only have a bare knowledge to how things actually work (firmware-related), but I'm interested in swapping my V4.2.2 board into an SKR Mini E3 V2.0 so that i can utilize its Linear Advance. Can I still use the Ext UI firmware as usual, or should i change some things, and if so what are those?

I'm not planning to swap my screen just yet, so I'm still sticking with the E3 V2's stock screen. A bit of guidance is really appreciated

**Comment by vhristenco** [comment, marked as answer]
You can use the V2 display on the SKR boards, but you have to build a custom cable and change some pins in the board configuration.
See [this thread](https://github.com/bigtreetech/BIGTREETECH-SKR-mini-E3/issues/424).

**Reply by raymond-arifianto** [reply]
Hm it really is a lot of hassle. In a scenario that i do change my screen to a TFT, would that mean that the Ext UI won't work anymore?

**Reply by vhristenco** [reply]
That's correct, ExtUI only works for the DWIN display.
